pkg/justification/processor_test.go

```diff
@@ -105,6 +110,14 @@ func TestCreateToken(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	mockFSClient, mockFSServer, mockFsCleanupFunc, err := testutil.NewMockFS(fakeProjectID)
```

Comment:
If we do a good job with encapsulating the `RemoteConfig`, then it seems we wouldn't need implement the whole fake firestore.